Java/fclUserManagement/src/main/java/com/fcl/adhocs/DatabaseConnect.java

```diff
@@ -0,0 +1,41 @@
+package com.fcl.adhocs;
+
+import java.sql.*;
+
+public class DatabaseConnect {
+    public static void main(String[] args) throws ClassNotFoundException {
+        // load the sqlite-JDBC driver using the current class loader
+        Class.forName("org.sqlite.JDBC");
+
+        Connection connection = null;
+        try {
+            // create a database connection
+            connection = DriverManager.getConnection("jdbc:sqlite:F:/Database/SQLite DB/mydb.db");
+            PreparedStatement statement = connection.prepareStatement("select * from person where name = 'leo'");
```

Comment:
Is this query used inside the App currently?
We need to remove the hardcoding here and make this ready for production